Application/HtmlSanitizer.hs

```diff
@@ -0,0 +1,82 @@
+module Application.HtmlSanitizer (sanitizeHtml, forbiddenTagNames,urlAttributes, AttributeName, AttributeValue) where
+
+
+import IHP.ViewPrelude
+import Text.HTML.TagSoup.Tree (parseTree, renderTree, TagTree(..))
+import Text.HTML.TagSoup (Tag(..))  
+import Data.Set (Set(), member, fromList)
+import Data.Text (unpack, isPrefixOf, toLower, pack)
+import Network.URI (uriScheme, parseURIReference, URI (..), isAllowedInURI, escapeURIString)
+import Codec.Binary.UTF8.String (encodeString)
+
+
+sanitizeHtml :: Text -> Text
```

Comment:
Seems quite custom. Was there no existing library to do it?

Reply:
yes, now we are using xss-sanitize